index.html

```diff
@@ -262,7 +262,7 @@ <h3 id="out-of-scope">Out of Scope</h3>
 					<p>The following features are out of scope, and will not be addressed by this Working Group group.</p>
 
 					<ul>
-						<li>Removal of any existing features defined by EPUB 3.2. Existing features will only be deprecated per existing practice.</li>
+						<li>Removal of any existing features defined by EPUB 3.2. Existing features will only be deprecated if they are not used by <a href="https://www.w3.org/TR/html-design-principles/#support-existing-content">existing EPUB 3 content</a> or unsupported by existing EPUB 3 reading systems.</li>
 						<li>Digital Rights Management (DRM).</li>
```

Comment:
While the reference to reading systems sounds fine to me, I am a bit worried by the language saying "they are not used by exiting EPUB3 content" which, if taken literally, is impossible to check. The link to the HTML design principles is a great idea but by just using the link the casual reader may simply miss it. Maybe it is better to make this reference more explicit in the text (even if the result is more verbose). Something like:

> Removal of any existing features defined by EPUB 3.2. Existing features will only be deprecated if their continued presence is not justified by the principles of supporting existing content (following the <a href="https://www.w3.org/TR/html-design-principles/#support-existing-content">principles layed out for HTML content </a>) or unsupported by existing EPUB 3 reading systems.

I am sure a native Anglo-saxon will find a better formulation...

Reply:
> I am sure a native Anglo-saxon will find a better formulation...

The language of the Anglo-Saxons is unintelligible to this uneducated American:
>Hwæt. We Gardena in geardagum,
>þeodcyninga, þrym gefrunon,
>hu ða æþelingas ellen fremedon.
>Oft Scyld Scefing sceaþena þreatum,
>monegum mægþum, meodosetla ofteah,
>egsode eorlas. Syððan ærest wearð
>feasceaft funden, he þæs frofre gebad,
>weox under wolcnum, weorðmyndum þah,
>oðþæt him æghwylc þara ymbsittendra
>ofer hronrade hyran scolde,
>gomban gyldan. þæt wæs god cyning. 

Anyway, I've updated the language a bit, making the source of the principle more clear without adding too much more verbiage.

Reply:
😀 I am fine with this text; I will contact Alan to see if things are fine with him...

Thx!